docs/naps/4-async-slicing.md

```diff
@@ -17,7 +17,7 @@ on the dimension slider positions and the current field of view.
 This project has two major goals.
 
 1. Slice layers asynchronously to keep napari responsive while allowing for slow slicing.
-2. Improve the logic of slicing layers to help developers better understand how it works.
+2. Improve the logic of slicing layers to help developers better understand how it works and unlock new feature development.
```

Comment:
Missed in last PR from comment: https://github.com/napari/napari/pull/4892/files#r935764844